MSVC/XP-Qt.bat

```diff
@@ -0,0 +1,16 @@
+@echo off
+setlocal
+set CURDIR=%~dp0
+set BASEDIR=%CURDIR%\..
+@echo on
+
+call "C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\VC\Auxiliary\Build\vcvarsall.bat" x86
+cd %BASEDIR%
+nmake distclean
+C:\Qt\5.10.1\msvc2015\bin\qmake.exe -spec win32-msvc || exit /b 1
```

Comment:
this line had to be changed for my build for some reason it installed at 5.10.1 inside a subfolder
`
C:\Qt\Qt5.10.1\5.10.1\msvc2015\bin\windeployqt --release output\XP-Qt\XP-Qt.exe || exit /b 1`